app/models/planning_application.rb

```diff
@@ -41,6 +41,7 @@ class WithdrawRecommendationError < RuntimeError; end
     has_many :notes, -> { by_created_at_desc }, inverse_of: :planning_application
     has_many :requests, class_name: "ValidationRequest"
     has_many :assessment_details, -> { by_created_at_desc }, inverse_of: :planning_application
+    has_many :permitted_development_rights
```

Comment:
Might be worth adding an order to this, just to make sure `#permitted_development_right` (singuler) always returns the right thing..?

Reply:
I believe it will return ordered by primary id which would be the created_at but no harm to add !